ui-tests/src/test/java/org/openmrs/reference/page/RegistrationPage.java

```diff
@@ -58,8 +59,14 @@
     static final By CANCEL = By.id("reviewSimilarPatients-button-cancel");
     private static final By CONFIRM_DATA = By.id("submit");
     private static final By PHONE_NUMBER_EDIT = By.xpath("//ul[@id='formBreadcrumb']/li/ul/li[2]/span");
+    private static final By SELECT_RELATIONSHIPTYPE = By.id("relationship_type");
+    private static final By PERSON_NAME = By.cssSelector("#relationship > p:nth-child(2) > input.person-typeahead.ng-pristine.ng-valid.ng-empty.ng-touched");
+    private static final By ADD_NEW_RELATIONSHIPTYPE = By.cssSelector("#relationship > p:nth-child(3) > a:nth-child(1) > i");
+    private static final By REMOVE_RELATIONSHIPTYPE = By.cssSelector("##relationship > p:nth-child(3) > a:nth-child(2) > i");
+    static final By RELATIONSHIPTYPE_CONFIRM = By.xpath(CONFIRMATION_DIV + "//div/div/p[6]");
     static By AUTO_LIST;
     private boolean acceptNextAlert = true;
+    private static final String NAME = "JOHN";
```

Comment:
Its better to set the String value within the test implementation than within the page. This would mean creating a generic method that can allow one to set the patient name within the test implementation .

The point here is to eliminate setting the String value within the page but rather creating a generic method that can set the name within any the test implementation that exercises the page functionality.